.ci/pytorch/build.sh

```diff
@@ -6,6 +6,12 @@ set -ex -o pipefail
 # (This is set by default in the Docker images we build, so you don't
 # need to set it yourself.
 
+# Source ROCm environment for theRock nightly builds (needed for docker exec)
+if [[ -f /etc/rocm_env.sh ]]; then
+  # shellcheck disable=SC1091
+  source /etc/rocm_env.sh
+fi
+
```

Comment:
Any sourcing of script to set required ROCm env vars should be done in `common.sh` since both `build.sh` and `test.sh` source it.